AnnoyancesFilter/Cookies/sections/cookies_specific.txt

```diff
@@ -277,6 +277,7 @@ foto-erhardt.com,foto-erhardt.de,seriouseats.com,fotogena.de,gqmagazine.fr,rossm
 !
 ! NOTE: Regular rules
 !
+lastdodo.de,lastdodo.nl,lastdodo.com,lastdodo.fr##.vc.show
```

Comment:
You rule is ok, but better to use the rule that related to cookies in the name, especially if it's possible.
```suggestion
lastdodo.de,lastdodo.nl,lastdodo.com,lastdodo.fr##div[data-controller="cookie-bar"]
```